### PR TITLE
Add text

### PR DESCRIPTION
**Pull request by awirb**
* Add a text box with max, mean and min value in difference plot
* Add option to filter qick plot data using a parameter of the simulation name, e.g. Mu, simulation type, ...
* Add outputformart option for saving figures as parameter

### CODE REVIEW

File: avaframe/runQuickPlot.py [hunk 1]
```diff
@@ -24,11 +24,14 @@
 log.info('Current avalanche: %s', avalancheDir)
 
 # REQUIRED+++++++++++++++++++
+# Which parameter to filter data, e.g. varPar = 'simType', values = ['null'] or varPar = 'Mu', values = ['0.055', '0.155']
+ # values need to be given as list, also if only one value
 outputVariable = ['pfd', 'ppr']
-simName = 'entres'
+values = ['null']
+varPar = 'simType'
```
Comment (fso42) [inline avaframe/runQuickPlot.py:31]: Would it make sense to include these in a ini file as default values?

**Review by fso42 — APPROVED** [review]

File: avaframe/out3SimpPlot/outQuickPlot.py [hunk 2]
```diff
@@ -109,13 +119,15 @@ def quickPlot(avaDir, suffix, cfg, simName):
         ax2.set_title(title)
         ax3 = fig.add_subplot(133)
         cmap = cmapdiv
-        im3 = plt.imshow(data1-data2, cmap=cmap,
+        im3 = plt.imshow(dataDiff, cmap=cmap,
                          extent=[0, Lx, 0, Ly], origin='lower', aspect=nx/ny)
         fig.colorbar(im3, ax=ax3)
+        ax3.text(nybox, nxbox, 'Mean: %.2f\n Max: %.2f\n Min: %.2f' % (diffMean, diffMax, diffMin), bbox=dict(boxstyle="square", ec='white', fc='white'),
+        horizontalalignment='left', verticalalignment='bottom')
```
Comment (matthiasto) [inline avaframe/out3SimpPlot/outQuickPlot.py:126]: could we maybe add some units here? to be sure if its m or kPa or %

**Review by matthiasto — CHANGES_REQUESTED** [review]
Could we add some doc to know what this does exactly and where to put the inputs, I am not sure I understood everything it does

File: avaframe/out3SimpPlot/outQuickPlot.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (awirb) [reply avaframe/out3SimpPlot/outQuickPlot.py:126]: the difference plot can be for pfd, ppr, pv, etc. and so far we also dont put units on the color bars

Comment (awirb) [reply avaframe/out3SimpPlot/outQuickPlot.py:126]: same as above, then i think we have to introduce the units for the colorbars too

[the remark this answers is not in the document]
Comment (matthiasto) [reply avaframe/out3SimpPlot/outQuickPlot.py:126]: okay but what appears on the 3rd plot is a value with the unit of what you look at? or some % ?
Just to be sure

File: avaframe/runQuickPlot.py (hunk 1 above)
Comment (awirb) [reply avaframe/runQuickPlot.py]: I added a outQuickPlotCfg.ini file with default values, and now the values and parameter used for filtering the data can be set there

File: avaframe/out3SimpPlot/outQuickPlot.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (awirb) [reply avaframe/out3SimpPlot/outQuickPlot.py]: I added the unit now

Comment (awirb) [reply avaframe/out3SimpPlot/outQuickPlot.py]: Added the unit

**Pull request merged**